gui/confirm.lua

```diff
@@ -0,0 +1,121 @@
+-- config ui for confirm
+
+confirm = require 'plugins.confirm'
+gui = require 'gui'
+widgets = require 'gui.widgets'
+
+Opts = defclass(Opts, widgets.Window)
+Opts.ATTRS = {
+    frame_title = 'Confirmation dialogs',
+    frame={w=36, h=17},
+}
+
+function Opts:init()
+    self:addviews{
+        widgets.List{
+            view_id = 'list',
+            frame = {t = 0, l = 0},
+            text_pen = COLOR_GREY,
+            cursor_pen = COLOR_WHITE,
+            choices = {},
+            on_submit = self:callback('toggle'),
+        },
+        widgets.HotkeyLabel{
+            frame = {b=1, l=0},
+            label='Toggle all',
+            key='CUSTOM_ALT_E',
+            on_activate=function() self:toggle_all(self.subviews.list:setSelected(i)) self:refresh() end,
```

Comment:
what's `i` set to here?
actually, looking at how this is used, this is supposed to be getSelected(), isn't it?

Reply:
Yeahhh, just fixed that up